### PR TITLE
added cosine function

### PR DESCRIPTION
**Pull request by KushagraVarshney05**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by KushagraVarshney05** [comment]
@yoiang @SSwiniarski i have completed the cos function.....pls review it!!!

**Review by SSwiniarski — APPROVED** [review]
@KushagraVarshney05 The PR itself looks good, but there are two other points:

1. You need to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=2244).
2. You made this PR off your main branch, which is not best practice. Next time make a new branch off of main for your PR. This shouldn't cause any issues as long as you don't submit another PR before this one is merged.

**Comment by KushagraVarshney05** [comment]
@SSwiniarski @yoiang  I have made the required changes

**Comment by KushagraVarshney05** [comment]
> @KushagraVarshney05 The PR itself looks good, but there are two other points:
> 
> 1. You need to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=2244).
> 2. You made this PR off your main branch, which is not best practice. Next time make a new branch off of main for your PR. This shouldn't cause any issues as long as you don't submit another PR before this one is merged.

i have signed the CLA but my pr still has not merged

**Comment by SSwiniarski** [comment]
> > @KushagraVarshney05 The PR itself looks good, but there are two other points:
> > 
> > 1. You need to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=2244).
> > 2. You made this PR off your main branch, which is not best practice. Next time make a new branch off of main for your PR. This shouldn't cause any issues as long as you don't submit another PR before this one is merged.
> 
> i have signed the CLA but my pr still has not merged

@KushagraVarshney05 Have patience. PRs require two reviews. After the next review and any issues are dealt with, then it can be merged.

File: content/php/concepts/math-functions/terms/cos/cos.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+```pseudo
+$result = cos($angle);
+```
+
+Where `$angle` is the angle in radians.
+
+## Codebyte Example
+
+Using `cos()` to return the tangent of the angle in radians:
```
Comment (KTom101) [inline content/php/concepts/math-functions/terms/cos/cos.md:28]:
```suggestion
Using `cos()` to return the cosine of the angle in radians:
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@KushagraVarshney05 I added on change

**Review by KTom101 — APPROVED** [review]
@KushagraVarshney05 I went ahead and committed the change. This looks good to merge. 👍

**Comment by KTom101** [comment]
@KushagraVarshney05 Also, please note for the future do not use your `main` branch. It is highly discouraged. Make a new branch for every new entry.

**Comment by KTom101** [comment]
@KushagraVarshney05 Entry is live: https://www.codecademy.com/resources/docs/php/math-functions/cos 

You're currently listed as an unknown contributor:

Screenshot 2023-06-17 at 5 52 26 PM

If you wish to remain that way, there is nothing more you need to do. Otherwise, if you want your profile listed follow the steps linked [here](https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036
).

**Pull request merged**